src/app/application.cpp

```diff
@@ -144,13 +144,15 @@ Application::Application(const QString &id, int &argc, char **argv)
     QPixmapCache::setCacheLimit(PIXMAP_CACHE_SIZE);
 #endif
 
+    m_portableModeEnabled = QDir(QCoreApplication::applicationDirPath()).exists(DEFAULT_PORTABLE_MODE_PROFILE_DIR);
```

Comment:
Please rename it to reflect its actual meaning: m_defaultProfileFolderFound. Current name confuses.

Reply:
Done.